### PR TITLE
MHR Fractional Text + UX Defaults

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#13521 & /bcgov/entity#14810

*Description of changes:*
* Updated Information text with regards to Fractional Data.
* Logic to include LCM default for NEW groups in MHRegistrations ONLY.
* Minor clean up and validation updates.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by ChasBoucher — APPROVED** [review]
Looks good.

File: ppr-ui/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ppr-ui",
-  "version": "0.4.0",
+  "version": "0.4.1",
```
Comment (dimak1) [inline ppr-ui/package.json]: Should be 0.4.2

**Review by dimak1 — APPROVED** [review]

**Review by doug-lovett — APPROVED** [review]
Looks good.

File: ppr-ui/package.json (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/package.json]: You bet, as with all the other pending PR's. Will update once the others are updated/in 👍

**Pull request merged**